cms/djangoapps/contentstore/views/item.py

```diff
@@ -294,6 +294,7 @@ def xblock_view_handler(request, usage_key_string, view_name):
 
         elif view_name in PREVIEW_VIEWS + container_views:
             is_pages_view = view_name == STUDENT_VIEW   # Only the "Pages" view uses student view in Studio
+            is_library_view = view_name == 'container_child_preview'
```

Comment:
If the view name is specific to libraries, then yes, I think it should be renamed.

But I also think that this CMS code should not have to know about libraries. Is there no way to keep the logic contained within libraries?